experimental/traffic-portal/src/app/shared/tp-header/tp-header.component.scss

```diff
@@ -53,6 +57,10 @@ nav {
 	}
 }
 
+a.mat-menu-item {
+	text-transform: uppercase;
```

Comment:
doesn't `mat-button` handle this for you?

Reply:
Ah, `mat-button` wouldn't do it for you anyway:

> According to the Material design spec button text has to be capitalized, however we have opted not to capitalize buttons automatically via `text-transform: uppercase`, because it can cause issues in certain locales. It is also worth noting that using ALL CAPS in the text itself causes issues for screen-readers, which will read the text character-by-character. We leave the decision of how to approach this to the consuming app.